### PR TITLE
Added a new annotation tutorial

### PR DESCRIPTION
**Pull request by sdorkenw**

### CODE REVIEW

File: FlyWireAnnotationTutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+    "\n",
+    "\n",
+    "## Authentication & Authorization\n",
+    "\n",
+    "If this is your first time to interact with any part of CAVE, chances are you need to setup your local credentials for your FlyWire account first. Please follow the section \"Setting up your credentials\" at the beginning of the tutorial above to do so.\n",
+    "\n",
+    "You will need to have access to the FlyWire's production dataset to retrieve annotations. Otherwise you will see\n",
+    "\n",
+    "```HTTPError: 403 Client Error: FORBIDDEN for url```\n",
+    "\n",
```
Comment (fcollman) [inline FlyWireAnnotationTutorial.ipynb:31]: This suggests on one line you need microns permissions, and the next that you need flywire permissssions

File: FlyWireAnnotationTutorial.ipynb [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+   "metadata": {},
+   "source": [
+    "## Reading annotaitons"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Annotations can be read directly from the annotaition service. However, no segment IDs will be assigned to the annotations through this interface. To access annotaions with segment IDs assigned, one must use the [materialization interface](https://github.com/seung-lab/CAVEclient/blob/master/FlyWireSynapseTutorial.ipynb).\n",
```
Comment (fcollman) [inline FlyWireAnnotationTutorial.ipynb:244]: typo

Comment (fcollman) [reply FlyWireAnnotationTutorial.ipynb:244]: in "annotaition"

File: FlyWireAnnotationTutorial.ipynb [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.5"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```
Comment (fcollman) [inline FlyWireAnnotationTutorial.ipynb:309]: I think your missing an explanation about what the user can expect to happen after they post the table, and how to query it.

File: FlyWireAnnotationTutorial.ipynb (hunk 1 above)
Comment (sdorkenw) [reply FlyWireAnnotationTutorial.ipynb:31]: I don't quite follow. There is just a disclaimer about the linked notebook
```
Please be advised that depending on your permission level you may not be able to execute all queries in this tutorial with the preset parameters as it was written with defaults for iarpa's microns project:
```

**Pull request merged**